todo_app/data/trello_items.py

```diff
@@ -0,0 +1,66 @@
+from flask import Flask, render_template, request
+import os
+import requests
+import json
+from .TrelloClass import Item
+
+
+
+def get_trello_items():
+    url = "https://api.trello.com/1/boards/620e338747c98f4449c740c7/lists"
```

Comment:
Although this id (and your list ids below) are not sensitive, it would still be nice to extract them so that your app is configurable for other people's Trello boards - currently we can _only_ run your app against your specific Trello board, rather than e.g. having different boards in different environments. Pulling those out into environment variables would be a nice touch (and similarly with any hard-coded list ids!)

```suggestion
    url = f"https://api.trello.com/1/boards/{os.getenv('TRELLO_BOARD_ID')}/lists"
```